File: 2114. Maximum Number of Words Found in Sentences/2114. Maximum Number of Words Found in Sentences.js
/**
 * @param {string[]} sentences
 * @return {number}
 */
var mostWordsFound = function(sentences) {
    var n=sentences.length
    var max=0
    for(let i=0;i<n;i++){
        var words=sentences[i].split(" ")
        var count=words.length
        if(count>max){
            max=count
        }
    }
    return max    
};

File: 2114. Maximum Number of Words Found in Sentences/2114. Maximum Number of Words Found in Sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int max=0;
        int count;
        for(int i=0; i<sentences.length; i++){
            String [] words=sentences[i].split(" ");
            count=words.length;
            if(count>max){
                max=count;
            }
        }
        return max;        
    }
}
